### ISSUE TITLE
[Bug] [rest-api] /hazelcast/rest/maps/running-jobs Request failed

### ISSUE THREAD

**Issue by gaotong521** [apache/seatunnel#5770]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

When rest-api is invoked to obtain the running information of any seatunnel (/hazelcast/rest/maps/running-jobs),seatunnel adopts cluster deployment and has three nodes in total. Only one node can successfully request and the other two nodes fail to request. Failure response :{"status":"fail", "message":"Please don't get coordinator service from an inactive master node"}

### SeaTunnel Version

2.3.3

### SeaTunnel Config

```conf
tcp-ip:
        enabled: true
        member-list:
          - 10.188.15.61
          - 10.188.15.62
          - 10.188.15.63
```

### Running Command

```shell
get 10.188.15.61:5801/hazelcast/rest/maps/running-jobs
```

### Error Exception

```log
2023-11-02 09:18:18,166 WARN  [.Log4j2HttpGetCommandProcessor] [hz.main.cached.thread-6] - [10.188.15.61]:5801 [seatunnel] [5.1] An error occurred while handling request HttpCommand [HTTP_GET]{uri='/hazelcast/rest/maps/running-jobs'}AbstractTextCommand[HTTP_GET]{requestId=0}
org.apache.seatunnel.engine.common.exception.SeaTunnelEngineException: Please don't get coordinator service from an inactive master node
        at org.apache.seatunnel.engine.server.SeaTunnelServer.getCoordinatorService(SeaTunnelServer.java:210) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at org.apache.seatunnel.engine.server.rest.RestHttpGetCommandProcessor.convertToJson(RestHttpGetCommandProcessor.java:242) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at org.apache.seatunnel.engine.server.rest.RestHttpGetCommandProcessor.lambda$handleRunningJobsInfo$2(RestHttpGetCommandProcessor.java:162) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193) ~[?:1.8.0_191]
        at java.util.Iterator.forEachRemaining(Iterator.java:116) ~[?:1.8.0_191]
        at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801) ~[?:1.8.0_191]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481) ~[?:1.8.0_191]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_191]
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708) ~[?:1.8.0_191]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_191]
        at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:510) ~[?:1.8.0_191]
        at org.apache.seatunnel.engine.server.rest.RestHttpGetCommandProcessor.handleRunningJobsInfo(RestHttpGetCommandProcessor.java:164) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at org.apache.seatunnel.engine.server.rest.RestHttpGetCommandProcessor.handle(RestHttpGetCommandProcessor.java:91) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at org.apache.seatunnel.engine.server.rest.RestHttpGetCommandProcessor.handle(RestHttpGetCommandProcessor.java:63) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at com.hazelcast.internal.ascii.TextCommandServiceImpl$CommandExecutor.run(TextCommandServiceImpl.java:402) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at com.hazelcast.internal.util.executor.CachedExecutorServiceDelegate$Worker.run(CachedExecutorServiceDelegate.java:217) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[?:1.8.0_191]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[?:1.8.0_191]
        at java.lang.Thread.run(Thread.java:748) [?:1.8.0_191]
        at com.hazelcast.internal.util.executor.HazelcastManagedThread.executeRun(HazelcastManagedThread.java:76) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
        at com.hazelcast.internal.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:102) ~[seatunnel-starter.jar:2.3.4-SNAPSHOT]
```

### Zeta or Flink or Spark Version

2.3.3

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by liugddx** [comment]
#5950

**Comment by gaotong521** [comment]
？？

**Comment by liugddx** [comment]
> ？？

This issue was fixed in #5950
